expression/builtin_miscellaneous_test.go

```diff
@@ -19,6 +19,35 @@ import (
 	"github.com/pingcap/tidb/util/types"
 )
 
+func (s *testEvaluatorSuite) TestIsIPv4(c *C) {
+	tests := []struct {
+		ip     string
+		expect interface{}
+	}{
+		{"192.168.1.1", 1},
```

Comment:
mysql 'is_ipv4' works differently with system call 'inet_aton'. I've fixed it and added related cases